deploy/openshift4/operator.nsx.vmware.com_v1_ncpinstall_cr.yaml

```diff
@@ -8,13 +8,16 @@ spec:
   # Note that if one node has multiple attached VirtualNetworkInterfaces, this function is not supported and should be set to false
   addNodeTag: true
   nsx-ncp:
-    # Uncomment below to add user-defined nodeSelector for NCP Deployment
-    #nodeSelector:
+    nodeSelector:
```

Comment:
Is it fine if `nodeSelector` is empty?

Reply:
Yes, it will be fine. In fact, it has the same effect as before.After applying ncp cr, k8s will recognize there is value defined for nodeSelector, so this option will be removed automatically. So, it only works with key: value defined.